packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -608,7 +608,12 @@ export const useNotifiSubscribe: ({
       let finalDiscordId = undefined;
 
       if (useDiscord === true) {
-        finalDiscordId = await client.createDiscordTarget('Default');
+        try {
+          finalDiscordId = await client.createDiscordTarget('Default');
+        } catch (e) {
+          // catch the case that the 'Default' discord already created
+          finalDiscordId = data.discordTargets[0].id;
+        }
```

Comment:
instead of a try/catch pattern, we should do a "search -> if not found then create" pattern

```
const existingDiscordTarget = data.discordTargets.find(target => target.name === 'Default');
let finalDiscordId: string | undefined = undefined;
if (existingDiscordTarget !== undefined) {
  finalDiscordId = existingDiscordTarget.id;
} else {
  const createdDiscordTarget = await client.createDiscordTarget('Default');
  finalDiscordId = createdDiscordTarget.id;
}
```

Reply:
Right!! it is much better solution. Thanks!